app/src/renderer/components/wallet/PageTransactions.vue

```diff
@@ -49,28 +42,21 @@ export default {
     ToolBar
   },
   computed: {
+    ...mapState(["transactions", "node"]),
     ...mapGetters([
       "filters",
-      "transactions",
+      "allTransactions",
       "wallet",
       "config",
       "delegation",
-      "connected"
+      "connected",
+      "validators"
     ]),
-    somethingToSearch() {
-      return !this.wallet.historyLoading && !!this.allTransactions.length
+    validatorURL() {
+      return "https://explorecosmos.network/validators"
```

Comment:
The link should be to the validators page in Voyager I think. Do you think linking to an outside page makes more sense?

Reply:
when i pass a `validatorURL` like `/staking/validators/` the way our voyager routing works, the `tendermint/ui` component forces the link into a new window as if it were `target="_blank"`. Probably fixed by using `vue-router` in the subcomponent but should we be using `vue-router` there?